erpnext/patches/v13_0/create_attendance_status_leave_type_abbr_and_set_remaining_half_day_status.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import unicode_literals
+import frappe
+from erpnext.hr.utils import create_standard_attendance_status
+
+def execute():
+
+    frappe.reload_doc("HR", "doctype", 'Attendance Status')
+    frappe.reload_doc("HR", "doctype", 'Leave Application')
+    frappe.reload_doc("HR", "doctype", 'Attendance')
```

Comment:
```suggestion
    frappe.reload_doc("HR", "doctype", "Attendance Status")
    frappe.reload_doc("HR", "doctype", "Leave Application")
    frappe.reload_doc("HR", "doctype", "Attendance")
    frappe.reload_doc("HR", "doctype", "Leave Type")
```

Patch failing. You will have to reload Leave Type too:

![image](https://user-images.githubusercontent.com/24353136/104883868-6da83b00-598b-11eb-9615-987e96166fa0.png)